capstone-frontend/src/components/CreateClub.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class CreateClub extends Component {
+  render() {
+    return (
+      <form action="https://8080-0b34ed39-12e2-4bb0-83f0-3edbd4365bbd.us-east1.cloudshell.dev/clubs" method="post">
+        <label for="name"> Club Name </label> <br />
+        <input type="text" id="name" name="name" /> <br />
+        <label for="description"> Club Description </label> <br />
+        <textarea rows="5" cols="75" type="text" id="description" name="description" /> <br />
+        <label for="ownerID"> ownerID </label> <br />
+        <input type="text" id="ownerID" name="ownerID" /> <br />
+        <label for="gbookID"> gbookID </label> <br />
+        <input type="text" id="gbookID" name="gbookID" /> <br />
+        <input type="submit" value="Submit" id="submit" /> <br />
```

Comment:
You should avoid the `/` tag in the <br> elements
https://google.github.io/styleguide/htmlcssguide.html#Document_Type

Reply:
This seems to give me an "Unterminated JSX contexts error"